config/_default/menus/main.en.yaml

```diff
@@ -4222,146 +4222,151 @@ menu:
       parent: tests_setup
       identifier: tests_setup_junit_xml
       weight: 107
+    - name: Network Traffic
```

Comment:
I understand that this is how it's called in Datadog Agent settings, but in the context of agentless mode it is not immediately clear what is the traffic that I as a customer should care about. Maybe "Network Settings" would be a better name?

Reply:
That makes sense, renamed.